src/components/Delegates/DelegateCard/DelegateActions.jsx

```diff
@@ -3,15 +3,26 @@ import { DelegateButton } from "./DelegateButton";
 import { DelegateSocialLinks } from "./DelegateSocialLinks";
 
 // TODO: add twitter and discord links (from delegate statement)
+const statement = {
+  discord: "agora",
+  twitter: "agora",
+};
 export function DelegateActions({ className, address, votingPower }) {
   return (
     <HStack
       justifyContent="space-between"
       alignItems="stretch"
-      className={className}
+      className={className ? className + "justify-between" : "justify-between"}
```

Comment:
Hi Kent, justify-content: space-between was either not working on HStack so I had to add this line